server/middleware/database.js

```diff
@@ -0,0 +1,18 @@
+const MongoClient = require("mongodb").MongoClient;
+require("dotenv").config;
+const database = "calendly";
+const collections = "test";
+const uri =
+  "mongodb+srv://team-lavender:lavender-team@cluster0.kenuz.mongodb.net/calendly?retryWrites=true&w=majority";
+// put the URI in process.env.ATLAS_URI, but not working for some reason
+const client = new MongoClient(uri, {
```

Comment:
if it doesn't work, how about using mongoose like below
mongoose
  .connect(process.env.<VARIABLE NAME>, { useNewUrlParser: true })
  .then(() => console.log("successfuly"))
  .catch((err) => console.log(err));